app/models/cloud_database_server.rb

```diff
@@ -0,0 +1,4 @@
+class CloudDatabaseServer < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
```

Comment:
@nasark we should add the `include NewWithTypeStiMixin` mixin here (allows creation of the subclass by passing in a `:type` attribute to the base-class' create)